pkg/blocks/ttps.go

```diff
@@ -67,6 +69,30 @@ type MitreAttack struct {
 	SubTechniques []string `yaml:"subtechniques,omitempty"`
 }
 
+// Targets represents the targets structure for a TTP.
+//
+// **Attributes:**
+//
+// Name: The name of the target.
+// Description: A description of the target.
+// Targets: A map of targets to be set for the TTP.
+type Targets struct {
+	OS    []string `yaml:"os,omitempty"`
+	Arch  []string `yaml:"arch,omitempty"`
+	Cloud []Cloud  `yaml:"cloud,omitempty"`
+}
+
+// Cloud represents the cloud provider structure for a TTP.
+//
+// **Attributes:**
+//
+// Provider: The name of the cloud provider to be targeted.
+// Region: The name of the cloud region to be targeted.
+type Cloud struct {
+	Provider string `yaml:"provider,omitempty"`
+	Region   string `yaml:"region,omitempty"`
```

Comment:
Region definitely feels like a regular CLI argument, not a "this TTP only works under these conditions" specifications like the rest of your targets - I wouldn't put it in here. If you leave it as a CLI argument, you can benefit from existing `args` machinery around default values for instance